### PR TITLE
Danielgent/ch220/mobile menu

### PR DESCRIPTION
**Pull request by danielgent**
https://user-images.githubusercontent.com/8496063/131154179-c5945099-0ffe-458b-86f9-b2315e164740.mp4

Language switcher will be done on _223/header-language-selector_

### CODE REVIEW

File: components/common/Navbar/Navbar.module.css [hunk 1]
```diff
@@ -3,30 +3,18 @@
 }
 
 .navContainer {
-  @apply bg-secondary;
+  @apply bg-secondary relative z-40;
```
[the remark this answers is not in the document]
Comment (frankjonatanschultz) [reply components/common/Navbar/Navbar.module.css:6]: 👍 i normally do that also. I guess we could set up named constants in tailwind.config https://tailwindcss.com/docs/z-index#customizing

[the remark this answers is not in the document]
Comment (danielgent) [reply components/common/Navbar/Navbar.module.css:6]: @frankjonatanschultz I created a ticket for this https://app.clubhouse.io/sesamy/story/243/set-constants-for-z-indexes

Let's discuss in the dev meeting?  As I don't see how we can make them named

*BUT* we could just create our own tailwind rules easily right? (I was reading through the docs after sending to my kindle and IIIRC I saw how to do this)

[the remark this answers is not in the document]
Comment (danielgent) [reply components/common/Navbar/Navbar.module.css:6]: @frankjonatanschultz Is there anything else blocking this task though? Just as the mobile header ticket is blocked by this and I could use some CSS help on that I think (I can make a Codepen demo before the meeting)

**Review by frankjonatanschultz — APPROVED** [review]

File: components/common/Navbar/Navbar.module.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (frankjonatanschultz) [reply components/common/Navbar/Navbar.module.css:6]: 👍 i'll have a look on the z-index stuff, not blocking this one, you can merge!

**Pull request merged**